### ISSUE TITLE
plugin.min.js scripts return http 502 error.

### ISSUE THREAD

**Issue by TimVee** [Budibase/budibase#8052]
**Hosting**
- Cloud
  - Tenant ID: Private app, I don't know if I can post it here

**Describe the bug**
When in the builder or in the app preview, budibase tries to to load custom components, but https://\<tenant ID\>.budibase.app/plugins/\<plugin name\>/plugins/\<plugin name\>/\<tenant ID\>/plugin.min.js returns a http 502 error.

I'm visiting budibase from Europe.

**To Reproduce**
1. Make sure you have a component plugin. I tried with both budibase-component-countup and budibase-component-star-rating
2. Go to the builder
3. Add a custom component from the plugin
4. Click preview

**Expected behavior**
The plugin scripts load, and the component shows up.

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Edge
 - Version: 105

**Comment by mike12345567** [comment]
Hi @TimVee - there was a problem with this - however if you update your app it should be resolved now.

You'll see an update is available within the app builder in the top right - it will appear as such:
![image](https://user-images.githubusercontent.com/4407001/193250054-d1571a7c-0a4b-485f-b556-7c100800c5d8.png)

**Comment by TimVee** [comment]
It says I'm on the latest version.

![msedge_D5uknOdbem](https://user-images.githubusercontent.com/70507163/193257327-045d9d33-34c1-4cde-87a7-e7fe40b87d8a.png)

**Comment by TimVee** [comment]
I got an update to 2.0.12, that didn't fix it either.

**Comment by TimVee** [comment]
2.0.14. I removed the plugin and uploaded it again, now I'm getting a 404 in the builder and 502 in the preview.

**Comment by mike12345567** [comment]
Re-opening this issue as it still seems to be occurring.

**Comment by TimVee** [comment]
Updated to 2.0.25, now the plugins showed, but I tried updating by uploading a version with an increased version number of the same name. That didn't work. I'm getting `last-modified: Wed, 05 Oct 2022 10:06:18 GMT` back from the server, so it seems to be a caching issue.
I tried deleting it and uploading with a different name, but the old version is still in the list of components to choose from when creating a new component in the builder, while the new one doesn't appear. Should I open a new issue for that?
